### PR TITLE
docs: add Biweekly Report (May 08 - May 22)

### PR DESCRIPTION
**Pull request by Qizeng-api7**
Fixes: #[Add issue number here]

Changes:

Screenshots of the change:

### CODE REVIEW

**Review by Yilialinn — DISMISSED** [review]

**Review by guoqqqi — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

File: blog/en/blog/2023/05/24/weekly-report-0524.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+"If you want to go fast, go alone. If you want to go far, go together." The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+
+We have also sorted out some issues for newcomers to the community to participate in! If you are interested, don't miss it!
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/05/24/yYk0cpo5_20230524.png)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2023/05/24/YL1CWg4i_new-contributors%20%281%29.png)
+
```
Comment (yzeng25) [inline blog/en/blog/2023/05/24/weekly-report-0524.md:29]: @Qizeng-api7 ,the number just does not make sense, I know some of the contributors on the list did not contribute in this period. Would you double-check and update if needed?

cc @SkyeYoung @SylviaBABY @lijing-21 @EstelleRao @tzssangglass @reeseliu715 @guitu168 

Update: @KID-G should be on this list with https://github.com/apache/apisix-website/pull/1568

Comment (yzeng25) [reply blog/en/blog/2023/05/24/weekly-report-0524.md:29]: 0a5afc4914e34a05c4f5dee2641e6bd

Comment (lijing-22) [reply blog/en/blog/2023/05/24/weekly-report-0524.md:29]: Please remove my name(lijing-21) from the list of contributors, thx!

Comment (SylviaBABY) [reply blog/en/blog/2023/05/24/weekly-report-0524.md:29]: After checking the list, I only reviewed a few PR this month and did not participate in other content. Secondly, @lunarwhite was my 2022 APISIX Open Source Summer Documentation Project student. He has not been contributing to the APISIX project for a long time, so please double-check everything

File: blog/zh/blog/2023/05/24/weekly-report-0524.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+
+## 贡献者统计
+
+![贡献者名单](https://static.apiseven.com/uploads/2023/05/24/yYk0cpo5_20230524.png)
+
+![新晋贡献者](https://static.apiseven.com/uploads/2023/05/24/YL1CWg4i_new-contributors%20%281%29.png)
+
```
Comment (yzeng25) [inline blog/zh/blog/2023/05/24/weekly-report-0524.md:27]: @Qizeng-api7 , ditto, please check and update the images soon.

File: blog/en/blog/2023/05/24/weekly-report-0524.md (hunk 1 above)
Comment (Yilialinn) [reply blog/en/blog/2023/05/24/weekly-report-0524.md:29]: I've checked the contributor list, you used the wrong time period 2022.05.08-2022.05.22, which should be 2023 @Qizeng-api7 https://github.com/nic-chen/github-statistic/pull/19

Comment (tzssangglass) [reply blog/en/blog/2023/05/24/weekly-report-0524.md:29]: pls check this list, my name(tzssangglass) should also be removed

Comment (yzeng25) [reply blog/en/blog/2023/05/24/weekly-report-0524.md:29]: Fixed in https://github.com/apache/apisix-website/pull/1596 and https://github.com/apache/apisix-website/pull/1597. @lijing-22 @SylviaBABY @tzssangglass @KID-G

Comment (reeseliu715) [reply blog/en/blog/2023/05/24/weekly-report-0524.md:29]: ...amazing

**Pull request merged**